toast-tk-plugin-loader/src/main/java/io/toast/tk/plugin/PluginLoader.java

```diff
@@ -79,11 +83,18 @@ private static void fillJarsList(
 			List<URL> jars,
 			File dir) {
 		try {
-			for (File jar : dir.listFiles(pathname -> pathname.isFile() && pathname.getName().endsWith(".jar"))) {
+			for (File jar : dir.listFiles(PluginLoader::isJar)) {
 				jars.add(jar.toURI().toURL());
 			}
 		} catch (Exception e) {
-			LOG.error(e.getMessage(), e);
+			LOG.error("Error to find jars in dir:" + dir + " - " + e.getMessage(), e);
 		}
 	}
+	
+	private static boolean isJar(File pathname){
+		LOG.error("debug:" + pathname);
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "debug:" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Use the built-in formatting to construct this argument. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2629)